### PR TITLE
Code fix

### PR DESCRIPTION
**Pull request by RaZeSloth**
Code fixes and one spell fix.

### CODE REVIEW

File: functions/docs.js [hunk 1]
```diff
@@ -13,9 +15,9 @@ fetch(`https://djsdocs.sorta.moe/v2/embed?src=${branch}&q=${encodeURIComponent(q
       .then(res => res.json())
       .then(json => {
         if (!json) return message.chanel.send(`:x: | No results for ${query}`)
-        message.channel.send({ embed: json });
+       return message.channel.send({ embed: json });
       })
       .catch(() => {
-        message.channel.send(`:x: | An unexpected err occrued`)
+        return message.channel.send(`:x: | An unexpected err accrued`)
```
Comment (reinacchi) [inline functions/docs.js]: Should be `occurred`, both were wrong.

**Review by reinacchi — CHANGES_REQUESTED** [review]

File: functions/non-public-functions/buttons.js [hunk 2]
```diff
@@ -21,6 +21,9 @@ module.exports = async function buttonss(content, options, client, message) {
         throw new Error('The buttons must be an array');
     }
 
+    /**
+     * @type {{ type: number; style: number | undefined; label: any; custom_id: any; emoji: { name: null; id: null; animated: boolean; }; url: any; }[]}
```
Comment (reinacchi) [inline functions/non-public-functions/buttons.js]: Is this an array of objects? If it is, I would recommend `Array<Object>` instead.

File: functions/docs.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
 const fetch = require("node-fetch");
+
+const { MessageEmbed } = require("discord.js")
 /**
  * Search the Discord.js Docs for a query
  * @param {String} query, query for the search
  * @param {String} branch, branch for the search
- * @param {String} Message, Discord Message 
- * @returns {Discord.MessageEmbed}
+ * @param {String} message, Discord Message
```
Comment (reinacchi) [inline functions/docs.js]: `message` should be `Discord.Message`

File: functions/docs.js (hunk 1 above)
Comment (RaZeSloth) [reply functions/docs.js]: What, I remember updating it? Well, yea. I'll fix it

**Review by reinacchi — APPROVED** [review]

**Comment by reinacchi** [comment]
@BIntelligent , please immediately merge this PR so that I could open another PR to fix the typings. It appears to be very outdated since no PRs were working on the typings.

I would like this to be merge as soon as possible to prevent files conflict 🙏🏻

**Pull request merged**